components/tools/OmeroPy/src/omero/gateway/__init__.py

```diff
@@ -1530,6 +1531,29 @@ def getMaxPlaneSize(self):
                 int(c.getConfigValue('omero.pixeldata.max_plane_height')))
         return self._maxPlaneSize
 
+    def getOrphanedImagesSettings(self):
+        default = {
+            "enabled": True,
+            "name": "Orphaned Images",
+            "description": ("This is a virtual container with "
+                            "orphaned images.")
+        }
```

Comment:
If there is such a requirement we can review all functions loading config in gateway and adjust them in another PR